lgc/state/PalMetadata.cpp

```diff
@@ -980,3 +951,182 @@ bool PalMetadata::fragmentShaderUsesMappedBuiltInInputs() {
   }
   return false;
 }
+
+// =====================================================================================================================
+// Returns the calling convention of the first hardware shader stage that will be executed in the pipeline.
```

Comment:
I know the following functions are handling graphics shader stages. Do we need to add a assert like this assert(isGraphics())

Reply:
I changed this function to handle CS.  Other functions already handle CS or will hit an llvm_unreachable.